### PR TITLE
First Draft of Async EP 2

### PR DESCRIPTION
**Pull request by lorecrafting**
Just put an Input and Button element in the index.

The README as compared to EP1 tells them to scrape data from the input field and the next step is a rather open ended: "Use the data from the API to display information in a presentable way, you will have to add some HTML elements in the index.html file" kind of thing.

And an additional "Handle errors from the API calls gracefully"

Should I add more requirements?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,27 @@
...
+
+1. Make a `/js` folder and create an `app.js` file in it.
+1. In the `index.html` file, load `app.js` via script tags.
+1. Use live-reload to serve up the files to your browser.
+1. Construct your XHR requests in the `app.js` file by scraping the value from the input field.
+1. A request to the SWAPI api should be initiated when the button is clicked.
+  Information from the data that comes back from the API should be displayed in
+  the body of the html.  Additional html elements should be created to properly
+  display and format the data in a presentable way.
+1. Think of an elegant way of handling errors.  For example, if the SWAPI API returns an error due to incorrect an incorrect URL fragment.
```
Comment (theRemix) [inline README.md]: Display any xhr errors in the dom, so the user can see.
for example, if a user inputs `99999` as an id, or `potato`
display the error on the page

**Review by theRemix — CHANGES_REQUESTED** [review]

**Review by theRemix — CHANGES_REQUESTED** [review]
add explanation of what *URL Fragment* is, 
maybe provide an example.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>The Async Force Ep 2</title>
+  <meta name="viewport" content="width=device-width" />
+</head>
+<body>
+  <h1>The Async Force Ep 2</h1>
+  <input id="url_fragment" type="text" placeholder="URL Fragment"/>
+  <button>Hit the Swapi API!</button>
```
Comment (theRemix) [inline index.html]: change the button to read "Get Person from SWAPI"

File: README.md [hunk 3]
```diff
@@ -1,2 +1,27 @@
...
+
+## Excercise
+*Remember to commit after each step.*
+
+1. Make a `/js` folder and create an `app.js` file in it.
+1. In the `index.html` file, load `app.js` via script tags.
+1. Use live-reload to serve up the files to your browser.
+1. Construct your XHR requests in the `app.js` file by scraping the value from the input field.
+1. A request to the SWAPI api should be initiated when the button is clicked.
+  Information from the data that comes back from the API should be displayed in
```
Comment (theRemix) [inline README.md]: add some more goals to the readme.
The developer should start by requesting the `/people/:id` api
then, describe what should be displayed
ex:

> Display this person's name
> Display all film titles that this person appeared in
> Display all planets this person has visited

**Review by theRemix — CHANGES_REQUESTED** [review]
add more instructions

**Comment by lorecrafting** [comment]
1. Added short explanation of URL fragment in hints section. 
2. Changed button name to "Get Person from SWAPI".
3. Reworded XHR error step.
4. Added additional data display requirements.
5 . Added a bonus step.

**Comment by lorecrafting** [comment]
Updated Readme:
Descoped some data display requirements.  Got rid of displaying extra fields that required nested API calls.  It was found that the complexity would take too long for students.

**Pull request merged**